libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteProgress.java

```diff
@@ -250,6 +279,9 @@ public RouteProgress build() {
   }
 
   public static Builder builder() {
-    return new AutoValue_RouteProgress.Builder();
+    return new AutoValue_RouteProgress.Builder()
+      .inTunnel(false)
```

Comment:
Yeah I understand the default is `false` - I added it because we aren't marking it as `NonNull` so `AutoValue` will throw an exception if we try to build a `RouteProgress` without the value in the builder.

Reply:
Nah, note that we're using a primitive so it can't be `null` https://github.com/mapbox/mapbox-navigation-android/blob/be0d05f89af7df8024e821558296dae45ff2d274/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteProgress.java#L161

I've tested it locally and didn't get any crashes. Am I missing something?

Reply:
@Guardiola31337 did you run the unit tests without it?  In the SDK, it should always be set in the processor, so that makes sense why you didn't see any crashes.  I can remove it and fix up the data in the tests - I guess we should be accounting for a crash in our unit tests with changes to production code.

Reply:
Yeah, makes sense. `AutoValue` ends up boxing everything in the object form and checking the properties, getting `IllegalStateException`s (`"Missing required properties:"`) if not initialized explicitly from our side (`builder()`). Thanks for clarifying 🙏